test/e2e-common.sh

```diff
@@ -281,8 +286,8 @@ function install_contour() {
 function install_knative_serving_standard() {
   readonly INSTALL_CERT_MANAGER_YAML="./third_party/cert-manager-${CERT_MANAGER_VERSION}/cert-manager.yaml"
 
-  echo ">> Creating knative-serving namespace if it does not exist"
-  kubectl get ns knative-serving || kubectl create namespace knative-serving
+  echo ">> Creating ${E2E_SYSTEM_NAMESPACE} namespace if it does not exist"
```

Comment:
That'd be bad if it existed :-)
Should there be a trap do _delete_ it in the end?

Reply:
I add `kubectl delete namespace` in the end of knative_teardown.